components/Session/Voting/Review/Content.js

```
@@ -0,0 +1,210 @@
+import styled from 'styled-components';
+import parse from 'html-react-parser';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+import ButterToast, { Cinnamon, POS_TOP, POS_RIGHT } from 'butter-toast';
+import _ from 'lodash';
+
+import NavLinks from '../Shared/NavLinks';
+import Thumbs from '../Shared/Thumbs';
+import Stats from '../Shared/Stats';
+
+const MarkdownIt = require('markdown-it');
+
+const MainContent = styled.div`
+  margin-top: 0rem;
+`;
+
+const SessionsContainer = styled.div`
+  .session-divider:last-of-type {
+    display: none;
+  }
+`;
+
+const SessionContainer = styled.div`
+  border-bottom: 1px solid ${({ theme }) => theme.colors.mediumGray};
+  padding-bottom: 3rem;
+
+  h4 {
+    margin-bottom: 1rem;
+  }
+
+  &:last-of-type {
+    border-bottom: none;
+  }
+
+  .thumbs-container {
+    margin-top: 0;
+  }
+`;
+
+const Section = styled.div`
+  margin-bottom: 3rem;
+  ul {
+    margin-top: 0;
+  }
+`;
+
+const DetailsHeader = styled.div`
+  font-size: 2rem;
+`;
+
+const GET_SESSIONS = gql`
+  query getVotedSessions($eventId: ID!) {
+    sessions {
+      me {
+        voting(eventId: $eventId) {
+          isVotingOpen
+          totalSubmitted
+          voted {
+            sessionId
+            title
+            longDescription
+            takeaways
+            vote
+          }
+        }
+      }
+    }
+  }
+`;
+
+const CAST_VOTE = gql`
+  mutation castVote($eventId: ID!, $vote: VoteInput!) {
+    sessions {
+      voting(eventId: $eventId) {
+        cast(vote: $vote) {
+          id
+          notes
+          memberId
+        }
+      }
+    }
+  }
+`;
+
+const Content = () => {
+  const {
+    loading: sessionsLoading,
+    error: sessionsError,
+    data: sessionsData,
+  } = useQuery(GET_SESSIONS, {
+    variables: { eventId: process.env.CURRENT_EVENT_ID },
+  });
+
+  if (sessionsLoading) return null;
+  if (sessionsError) return null;
```

Instead of returning null here you can use `throw sessionsError` so Sentry can catch it